modules/nf-core/bbmap/bbduk/environment.yml

```diff
@@ -4,4 +4,4 @@ channels:
   - bioconda
   - defaults
 dependencies:
-  - bioconda::bbmap=39.01
+  - bioconda::bbmap=38.18
```

Comment:
This is moving back a version for conda, but the singularity/docker versions is still `39.01`? Should these all be made consistent? Was this to get tests to work with conda?

Reply:
This explains why their md5s are clashing! Yes let me move back the others.